docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -370,7 +460,7 @@ To search for remote Bluetooth devices:
 
 
 <!-- TODO #03:
```

Comment:
This TODO comment can be removed.

Reply:
All TODO comments will be removed in the last commit.